plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/ChatController.kt

```diff
@@ -103,6 +112,33 @@ class ChatController private constructor(
         authController = AuthController(),
     )
 
+    init {
```

Comment:
why don't we put the initialization at projectStartupActivity (`CodeWhispererProjectStartupActivity`)

Reply:
`projectStartupActivity` should be guranteed by IDE to execute only once

Reply:
Good let me do this change

Reply:
isee, let's create a different projectStartupActivity under chat package

Reply:
since chat module now is referencing codewhisperer module as dependency https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/chat/jetbrains-community/build.gradle.kts#L21

Reply:
so we can just add one under chat
```
class ChatProjectStartupActivity : StartupActivity.DumbAware {
    private var projectContextServiceInit = false

    override fun runActivity(project: Project) {
           // initialization
    }
```

Reply:
Let me give this a try! Thank you!

Reply:
I moved it to Q start up activity and it works!